src/encoder.rs

```diff
@@ -524,7 +524,7 @@ impl<T: Pixel> FrameInvariants<T> {
       sequence.sb_size_log2(),
       config.width,
       config.height,
-      config.tile_cols_log2.min(min_tile_cols),
+      config.tile_cols_log2.max(min_tile_cols),
```

Comment:
IMO, the caller should only provide `config.tile_cols_log2`, it's the resposability of `TilingInfo` internals to clip the requested value if necessary (it already does that for other constraints).

Moreover, it seems wrong to min (or max) a number of cols in log2 with a number of cols (not in log2).

Reply:
yeah it looks like that part of the condition was always wrong. I'll move it inside TilingInfo and fix it.

Reply:
In fact you already wrote the exact same condition in TilingInfo, so I just deleted this one entirely.